PatientRegistrationApp/src/components/GreetingPage.vue

```diff
@@ -52,10 +50,8 @@
 									<v-icon left color="secondary">mdi-account</v-icon>
 									Register myself
 							</v-btn>
-						</template>
-						<span>Register yourself!</span>
-					</v-tooltip>
-					<v-tooltip top>
+
+					<v-tooltip right :nudge-bottom='130' max-width="20%">
```

Comment:
Can you anchor the text to the center bottom of the option, top middle of the tool-tip?